src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -28,6 +29,9 @@ public class OverseasEntitySubmissionDto {
     @JsonProperty(ENTITY_NAME_FIELD)
     private String entityName;
 
+    @JsonProperty(ENTITY_NUMBER_FIELD)
+    private String entityNumber;
```

Comment:
we will also need to update the schema.json file (in the spec folder) when we add/change the model structure

Reply:
done